child_compassion/views/project_reservation_view.xml

```diff
@@ -21,6 +21,7 @@
                         <group>
                             <group>
                                 <field name="channel_name" />
+                                <field name="child_id" />
```

Comment:
You can add attributes `attrs="{'invisible': [('child_id', '=', False)]}"` to hide it if not set.